src/libtorch.cc

```diff
@@ -552,11 +582,48 @@ ModelInstanceState::Create(
 
 ModelInstanceState::ModelInstanceState(
     ModelState* model_state, TRITONBACKEND_ModelInstance* triton_model_instance)
+#ifdef TRITON_ENABLE_GPU
+    : BackendModelInstance(model_state, triton_model_instance),
+      model_state_(model_state), device_(torch::kCPU), is_dict_input_(false),
+      pytorch_stream_(at::cuda::getDefaultCUDAStream())
```

Comment:
Why is 'pytorch_stream_' initialization is needed here? What happens if `pytorch_stream_` remains uninitialized for CPU instances?

Reply:
If important, can we not set `pytorch_stream_=at::cuda::getDefaultCUDAStream()` within the function below? It will avoid a lot of code duplication.

Reply:
See my response here: https://github.com/triton-inference-server/pytorch_backend/pull/73#discussion_r930512176

Reply:
I have fixed the duplication issue.